block/stream/output/state_changes.proto

```diff
@@ -63,45 +62,6 @@ import "state/token/token.proto";
 import "state/token/token_relation.proto";
 import "timestamp.proto";
 
-/**
- * The "cause" of a state change.
- *
- * What was the network doing that led to the change in state?<br/>
- * The most common event is, of course, a user transaction.
- * Other sources of state changes include end-of-block housekeeping changes,
- * data migration, and "system" transactions.
- */
-enum StateChangesCause {
```

Comment:
I'm sure there's a conversation I missed, but since I missed it–why are we removing `StateChangesCause`?

Reply:
It's unnecessary.  We don't really need the block node to know what caused a state change, we just need to record the state change and update the block-node copy of the state Merkle tree.

Reply:
Most of these changes are discussed on the HIP discussion page, if you're interested.

Reply:
What about state changes that don't come as a result of a specific transaction, e.g. the running hashes? I'm not sure we'll be able to pin the state changes currently with `CAUSE_END_OF_BLOCK` to anything concrete without the cause..

Reply:
The issue is that nobody really cares to know a cause.
It can be determined if an entity is _interested_ (I can describe how separately if needed), but it's not material to the block stream or block node.